src/sql-parser/src/parser.rs

```diff
@@ -4036,14 +4276,17 @@ impl<'a> Parser<'a> {
     }
 
     /// Parse a copy statement
-    fn parse_copy(&mut self) -> Result<Statement<Raw>, ParserError> {
+    fn parse_copy(&mut self) -> Result<Statement<Raw>, ParserStatementError> {
```

Comment:
Why is this changing to a `ParserStatementError`? Internally it's mapping to a no-statement error so it seems like this should stay as a `ParserError` and the caller should call `map_parser_error(StatementKind::Copy)`?

Reply:
Internally there are statements being parsed (e.g., `COPY` vs `SUBSCRIBE`). I think the nested statement failing to parse is more intersting/applicable.